ckanext/datatablesview/templates/datatables/datatables_view.html

```diff
@@ -0,0 +1,51 @@
+{% extends "base.html" %}
+
+{% block page %}
+  <table id="dtprv"
+      class="table table-striped table-bordered"
+      data-module="datatables_view"
+      data-paging="true"
+      data-server-side="true"
+      data-processing="true"
+      data-ajax='{
+          "url": "/datatables/ajax/{{ resource_view.id }}",
+          "type": "POST",
+          "data": { "filters": "{{ request.params.get('filters', '')|e }}" }
+        }'
+      {% if resource_view.get('responsive') %}
+        data-responsive="true"
+        data-scroll-x="false"
+      {% else %}
+        data-responsive="false"
+        data-scroll-x="true"
+      {% endif %}
+      data-fixed-header="true"
+      data-fixed-columns="true"
+      data-dom='"Blifrtip"'
+      data-buttons='[
```

Comment:
Is this param being used? I can't find the "Hide/Unhide Columns" option anywhere ( I suggest we use "Show" if it actually displayed). Also where are the options for CSV and PDF if `export_buttons` is True?